lib/WebSocket.js

```diff
@@ -22,6 +22,7 @@ const Sender = require('./Sender');
 
 const closeTimeout = 30 * 1000; // Allow 30 seconds to terminate the connection cleanly.
 const protocolVersion = 13;
+const supportedProtocolVersions = [8, 13];
```

Comment:
Would you mind merging this line with the previous one?

```js
const protocolVersions = [8, 13];
```

and use 

```js
protocolVersion: protocolVersions[1]
```

on line 510?